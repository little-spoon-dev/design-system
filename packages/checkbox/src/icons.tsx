import { shadeBlack, shadeWhite } from '@littlespoon/theme/lib/colors/token'

export const checkedIcon = (
  <svg width="16" height="16" viewBox="0 0 18 18" fill="none" xmlns="http://www.w3.org/2000/svg">
    <title>Checked checkbox</title>
    <desc>Square with checkmark</desc>
    <path
      d="M16 0H2C0.89543 0 0 0.89543 0 2V16C0 17.1046 0.89543 18 2 18H16C17.1046 18 18 17.1046 18 16V2C18 0.89543 17.1046 0 16 0Z"
      fill={shadeBlack}
    />
    <path
      d="M14.4 5.2876L6.97498 12.7126L3.59998 9.3376"
      stroke={shadeWhite}
      strokeWidth="1.5"
      strokeLinecap="round"
      strokeLinejoin="round"
    />
  </svg>
)

export const uncheckedIcon = (
  <svg width="16" height="16" viewBox="0 0 18 18" fill="none" xmlns="http://www.w3.org/2000/svg">
    <title>Unchecked checkbox</title>
    <desc>Empty square</desc>
    <path
      d="M2 0.916748H16C16.6904 0.916748 17.25 1.47639 17.25 2.16675V16.1667C17.25 16.8571 16.6904 17.4167 16 17.4167H2C1.30964 17.4167 0.75 16.8571 0.75 16.1667V2.16675C0.75 1.47639 1.30964 0.916748 2 0.916748Z"
      stroke={shadeBlack}
      strokeWidth="1.5"
      strokeLinecap="round"
      strokeLinejoin="round"
    />
  </svg>
)
