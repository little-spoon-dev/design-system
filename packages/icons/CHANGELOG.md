# Change Log

All notable changes to this project will be documented in this file.
See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.

## [2.0.5](https://github.com/little-spoon-dev/design-system/compare/@littlespoon/icons@2.0.4...@littlespoon/icons@2.0.5) (2022-09-21)

### Bug Fixes

- info icon issue ([2615fd6](https://github.com/little-spoon-dev/design-system/commit/2615fd6e6e2f5c376df3d4ceecdd7ff06e84b680))

## [2.0.4](https://github.com/little-spoon-dev/design-system/compare/@littlespoon/icons@2.0.3...@littlespoon/icons@2.0.4) (2022-09-09)

**Note:** Version bump only for package @littlespoon/icons

## [2.0.3](https://github.com/little-spoon-dev/design-system/compare/@littlespoon/icons@2.0.2...@littlespoon/icons@2.0.3) (2022-09-06)

### Bug Fixes

- **icons:** fix testing failure issue ([2259ba5](https://github.com/little-spoon-dev/design-system/commit/2259ba5bf6ca9cadc3f46516f178a8ba6ddf9fa5))
- **icons:** update add icon ([7f2ec6d](https://github.com/little-spoon-dev/design-system/commit/7f2ec6df5b15d69234b692cd7e589b365f001b29))

## [2.0.2](https://github.com/little-spoon-dev/design-system/compare/@littlespoon/icons@2.0.1...@littlespoon/icons@2.0.2) (2022-09-06)

### Bug Fixes

- **icons:** add exports of new PCH icons LS-3975 ([8320710](https://github.com/little-spoon-dev/design-system/commit/8320710e428bab89663add96b56b211a733e4370))

## [2.0.1](https://github.com/little-spoon-dev/design-system/compare/@littlespoon/icons@2.0.0...@littlespoon/icons@2.0.1) (2022-09-01)

**Note:** Version bump only for package @littlespoon/icons

# [2.0.0](https://github.com/little-spoon-dev/design-system/compare/@littlespoon/icons@1.9.0...@littlespoon/icons@2.0.0) (2022-09-01)

### Bug Fixes

- **icons:** fixed add icon test LS-3568 ([244523f](https://github.com/little-spoon-dev/design-system/commit/244523f4a4ac46e61593fa4f9b62677aaf16de30))
- **icons:** fixed logout icon test LS-3568 ([05f353f](https://github.com/little-spoon-dev/design-system/commit/05f353fbb49b9de985a4991e6246be14e0838717))
- **icons:** wrong icon name in ui theme LS-3568 ([9565b2b](https://github.com/little-spoon-dev/design-system/commit/9565b2b6cf6b6b5e6569c919b1db2933cb29e893))

### Features

- **icons:** account icon with story and test ([28fc03b](https://github.com/little-spoon-dev/design-system/commit/28fc03b687d1020ffa294dd2e4a83ca85956a1cd))
- **icons:** add size prop to close icon LS-3568 ([3f9ac8c](https://github.com/little-spoon-dev/design-system/commit/3f9ac8c454cf56250f080fc159c56107b2d7305b))
- **icons:** bill icon with story and test LS-3568 ([173ff41](https://github.com/little-spoon-dev/design-system/commit/173ff41451ca0514df72dcbef24a49ac55704cbd))
- **icons:** box icon with story and test LS-3568 ([99638ef](https://github.com/little-spoon-dev/design-system/commit/99638ef77c6f21a2e7cca24f27562ba9baa3857d))
- **icons:** calendar icon with story and test ([203b485](https://github.com/little-spoon-dev/design-system/commit/203b485b5154a68a9a555f04a25c48201dddfb7d))
- **icons:** caret icon with story and test ([1dd1558](https://github.com/little-spoon-dev/design-system/commit/1dd15586b3811f167213a64433b7647786dda5e1))
- **icons:** cart icon with story and test LS-3568 ([cd37957](https://github.com/little-spoon-dev/design-system/commit/cd37957a4d8d1d6bda63bead5f691a6ba8ca2286))
- **icons:** gift card icon with story and test ([87a8a71](https://github.com/little-spoon-dev/design-system/commit/87a8a71115e0f40e99e57aafa1ea64e215209111))
- **icons:** hamburger menu icon with story and test ([5faa577](https://github.com/little-spoon-dev/design-system/commit/5faa5775f62de0065fb29ddc2e3239cf3a911a23))
- **icons:** logout icon with story and test ([3c516f1](https://github.com/little-spoon-dev/design-system/commit/3c516f1d6822d45d4144e7a93af3da17ec87ad24))
- **icons:** rename open icon to add icon add size prop to it LS-3568 ([8816d18](https://github.com/little-spoon-dev/design-system/commit/8816d189a7d16ba5737ed68d3684fff6ad8a57a1))

### BREAKING CHANGES

- **icons:** rename OpenIcon to AddIcon

## [1.9.0](https://github.com/little-spoon-dev/design-system/compare/@littlespoon/icons@1.8.0...@littlespoon/icons@1.9.0) (2022-08-30)

### Bug Fixes

- **icons:** added explicit type and ([0974d75](https://github.com/little-spoon-dev/design-system/commit/0974d75372671793f920c5a71db0ff5784c2eca1))
- **icons:** fixed helpers props types ([69331db](https://github.com/little-spoon-dev/design-system/commit/69331db577638f5b62942b77f867054a272ebc51))

### Features

- **icons:** add utils/css-helpers for getting CSS ([1e15434](https://github.com/little-spoon-dev/design-system/commit/1e154347ee5747da4bebe7ded2c4ad483883aab7))

## [1.8.0](https://github.com/little-spoon-dev/design-system/compare/@littlespoon/icons@1.7.0...@littlespoon/icons@1.8.0) (2022-08-25)

### Bug Fixes

- **icons:** fix svg alignment issue ([2f762c1](https://github.com/little-spoon-dev/design-system/commit/2f762c17b4a56746dd4328e9ec3916030c25ebe5))

### Features

- **icons:** create subtract icon ([6e26f75](https://github.com/little-spoon-dev/design-system/commit/6e26f75ae7eb5a81b0232d3a63a01f38b66fedfd))

## [1.7.0](https://github.com/little-spoon-dev/design-system/compare/@littlespoon/icons@1.6.2...@littlespoon/icons@1.7.0) (2022-08-17)

### Features

- **icons:** add checkmark and exclamation mark icons LS-3819 ([d7d8937](https://github.com/little-spoon-dev/design-system/commit/d7d89374d34ce337c9a81365f75dc0f0b5e14fb9))

## [1.6.2](https://github.com/little-spoon-dev/design-system/compare/@littlespoon/icons@1.6.1...@littlespoon/icons@1.6.2) (2022-08-11)

**Note:** Version bump only for package @littlespoon/icons

## [1.6.1](https://github.com/little-spoon-dev/design-system/compare/@littlespoon/icons@1.6.0...@littlespoon/icons@1.6.1) (2022-07-15)

**Note:** Version bump only for package @littlespoon/icons

## [1.6.0](https://github.com/little-spoon-dev/design-system/compare/@littlespoon/icons@1.5.6...@littlespoon/icons@1.6.0) (2022-04-12)

### Features

- **all:** add react 18 to package.json peerDependencies ([010aeb4](https://github.com/little-spoon-dev/design-system/commit/010aeb4320c92dd1747093904b0d82c7743eb8e8))

## [1.5.6](https://github.com/little-spoon-dev/design-system/compare/@littlespoon/icons@1.5.5...@littlespoon/icons@1.5.6) (2021-11-13)

**Note:** Version bump only for package @littlespoon/icons

## [1.5.5](https://github.com/little-spoon-dev/design-system/compare/@littlespoon/icons@1.5.4...@littlespoon/icons@1.5.5) (2021-11-13)

**Note:** Version bump only for package @littlespoon/icons

## [1.5.4](https://github.com/little-spoon-dev/design-system/compare/@littlespoon/icons@1.5.3...@littlespoon/icons@1.5.4) (2021-10-28)

**Note:** Version bump only for package @littlespoon/icons

## [1.5.3](https://github.com/little-spoon-dev/design-system/compare/@littlespoon/icons@1.5.2...@littlespoon/icons@1.5.3) (2021-10-28)

**Note:** Version bump only for package @littlespoon/icons

## [1.5.2](https://github.com/little-spoon-dev/design-system/compare/@littlespoon/icons@1.5.1...@littlespoon/icons@1.5.2) (2021-10-27)

**Note:** Version bump only for package @littlespoon/icons

## [1.5.1](https://github.com/little-spoon-dev/design-system/compare/@littlespoon/icons@1.5.0...@littlespoon/icons@1.5.1) (2021-10-22)

**Note:** Version bump only for package @littlespoon/icons

## [1.5.0](https://github.com/little-spoon-dev/design-system/compare/@littlespoon/icons@1.4.0...@littlespoon/icons@1.5.0) (2021-10-20)

### Bug Fixes

- **icons:** add open icon ([23233f9](https://github.com/little-spoon-dev/design-system/commit/23233f9e10f151e4a5ebd597211270094e8b0d62))
- **icons:** name in test ([1c3bdbc](https://github.com/little-spoon-dev/design-system/commit/1c3bdbcdf8a9cc84db8c644deab83053b20f61da))
- **icons:** pr comments ([5527d6b](https://github.com/little-spoon-dev/design-system/commit/5527d6b2e2977f2c77bb4bc98e6e000512a77d6b))
- **icons:** remove desc ([594b777](https://github.com/little-spoon-dev/design-system/commit/594b7777f88efb9a494630d1d3f34ae0b0b03c36))
- rotate ([9b2f7db](https://github.com/little-spoon-dev/design-system/commit/9b2f7db1a9e9c360ea94d215ae2d001767bd8607))

### Features

- **icons:** new open icon ([b6a2cb5](https://github.com/little-spoon-dev/design-system/commit/b6a2cb5dd358d9c31d403eceee5e25db3c54eb79))

## [1.4.0](https://github.com/little-spoon-dev/design-system/compare/@littlespoon/icons@1.3.0...@littlespoon/icons@1.4.0) (2021-10-19)

### Bug Fixes

- **icons:** fix transform style for ArrowIcon LS-2041 ([961e7d9](https://github.com/little-spoon-dev/design-system/commit/961e7d91c3a58f68a014fdab5f0747137b4f11fd))

### Features

- **icons:** add prop fill to FilterIcon LS-2041 ([7d717dc](https://github.com/little-spoon-dev/design-system/commit/7d717dc87bbf65dc45b613ac3b945a6fd10fceb5))
- **icons:** add props fill and stroke to CloseIcon LS-2041 ([513b43d](https://github.com/little-spoon-dev/design-system/commit/513b43d55b7c3ce9cbb0c428239a1e9f1b2a7ce1))

## [1.3.0](https://github.com/little-spoon-dev/design-system/compare/@littlespoon/icons@1.2.0...@littlespoon/icons@1.3.0) (2021-10-19)

### Features

- **icons:** add filter icon LS-1943 ([33f0fd5](https://github.com/little-spoon-dev/design-system/commit/33f0fd51cce0c6bf9bcff07ea88ea400e6c2d994))
- **icons:** cleaning up filter icon props LS-1943 ([deaf0a9](https://github.com/little-spoon-dev/design-system/commit/deaf0a916d0268969fe7851cc109cfe8a1c73798))
- **icons:** linting after merge LS-1943 ([be94c97](https://github.com/little-spoon-dev/design-system/commit/be94c971f5a90250060f2a2882a6843c8d872429))
- **icons:** reverting manual icons version bump LS-1943 ([64d7009](https://github.com/little-spoon-dev/design-system/commit/64d7009f5f1344337c5492f3264aff7d6c4db143))
- **icons:** switching filter icon to rem LS-1943 ([1b55436](https://github.com/little-spoon-dev/design-system/commit/1b5543601af64499b5e9e98381af46ccf9b384fe))
- **icons:** tweaking filter icon based on feedback LS-1943 ([0af2583](https://github.com/little-spoon-dev/design-system/commit/0af25839ac592b2a10c9a4026aafc38f73b4a2ce))

## [1.2.0](https://github.com/little-spoon-dev/design-system/compare/@littlespoon/icons@1.1.5...@littlespoon/icons@1.2.0) (2021-10-18)

### Features

- **icons:** add CloseIcon LS-1944 ([032e01c](https://github.com/little-spoon-dev/design-system/commit/032e01c02d738505851a16fdd33814853130ac25))

## [1.1.5](https://github.com/little-spoon-dev/design-system/compare/@littlespoon/icons@1.1.4...@littlespoon/icons@1.1.5) (2021-10-13)

**Note:** Version bump only for package @littlespoon/icons

## [1.1.4](https://github.com/little-spoon-dev/design-system/compare/@littlespoon/icons@1.1.3...@littlespoon/icons@1.1.4) (2021-10-13)

**Note:** Version bump only for package @littlespoon/icons

## [1.1.3](https://github.com/little-spoon-dev/design-system/compare/@littlespoon/icons@1.1.2...@littlespoon/icons@1.1.3) (2021-10-13)

**Note:** Version bump only for package @littlespoon/icons

## [1.1.2](https://github.com/little-spoon-dev/design-system/compare/@littlespoon/icons@1.1.1...@littlespoon/icons@1.1.2) (2021-10-04)

**Note:** Version bump only for package @littlespoon/icons

## [1.1.1](https://github.com/little-spoon-dev/design-system/compare/@littlespoon/icons@1.1.0...@littlespoon/icons@1.1.1) (2021-10-04)

**Note:** Version bump only for package @littlespoon/icons

## 1.1.0 (2021-09-27)

### Features

- **icons:** add arrow icon package LS-1817 ([2bca572](https://github.com/little-spoon-dev/design-system/commit/2bca57204c63d5b0392499c3e396dd8314c96565))
- **icons:** add direction prop to ArrowIcon component ([37bfd6e](https://github.com/little-spoon-dev/design-system/commit/37bfd6e6c499abbbbb0b458c45eff34f202d37c1))
