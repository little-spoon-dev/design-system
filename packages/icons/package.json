{
  "name": "@littlespoon/icons",
  "version": "1.6.0",
  "description": "Little Spoon icons",
  "main": "lib/index.js",
  "scripts": {
    "build": "tsc",
    "build:flowtypes": "flowgen lib && rm -rf lib && mv exports lib && yarn build",
    "clean": "rm -rf coverage exports lib",
    "lint": "eslint .",
    "lint:fix": "yarn lint --fix",
    "lint:tsc": "tsc --noEmit",
    "test": "jest",
    "test:watch": "jest --watch"
  },
  "dependencies": {
    "@littlespoon/theme": "1.12.0"
  },
  "peerDependencies": {
    "react": "17 || 18",
    "styled-components": "4 || 5"
  },
  "repository": {
    "type": "git",
    "url": "https://github.com/little-spoon-dev/design-system.git",
    "directory": "packages/icons"
  },
  "bugs": {
    "url": "https://github.com/little-spoon-dev/design-system/issues"
  },
  "keywords": [
    "littlespoon",
    "icons"
  ],
  "files": [
    "lib"
  ],
  "publishConfig": {
    "access": "public"
  },
  "license": "UNLICENSED"
}
