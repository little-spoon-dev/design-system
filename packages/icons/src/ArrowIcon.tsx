import { shadeBlack } from '@littlespoon/theme/lib/colors/token'

import { ArrowBase } from './ArrowBase'

export interface ArrowIconProps extends React.SVGAttributes<SVGElement> {
  /**
   * The fill of the component. Defaults to black.
   */
  fill?: string

  /**
   * The direction of the component. Defaults to "down".
   */
  direction?: 'up' | 'down' | 'left' | 'right'

  /**
   * The size of the component. Defaults to "small".
   */
  size?: 'xsmall' | 'small' | 'medium' | 'large'
}

export default function ArrowIcon({
  direction = 'down',
  fill = shadeBlack,
  size = 'small',
  ...other
}: ArrowIconProps): React.ReactElement {
  return (
    <ArrowBase
      xmlns="http://www.w3.org/2000/svg"
      width="15"
      height="10"
      viewBox="0 0 15 10"
      size={size}
      direction={direction}
      {...other}
    >
      <title>Arrow icon</title>
      <desc>Arrow icon</desc>
      <path
        fill={fill}
        fillRule="evenodd"
        d="M7.754 9.392l6.644-6.861a.365.365 0 000-.507L13.027.608a.353.353 0 00-.508 0L7.5 5.791 2.481.608a.353.353 0 00-.508 0L.603 2.024a.365.365 0 000 .507l6.643 6.86c.14.145.368.145.508 0z"
      />
    </ArrowBase>
  )
}
