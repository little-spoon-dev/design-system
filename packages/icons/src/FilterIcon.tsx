import { shadeBlack, shadeWhite } from '@littlespoon/theme/lib/colors/token'
import styled from 'styled-components'

import { getSvgSize } from './utils/css-helpers'

export interface FilterIconProps extends React.SVGAttributes<SVGElement> {
  /**
   * The fill of the component. Defaults to white.
   */
  fill?: string

  /**
   * The size of the component. Defaults to "small".
   */
  size?: 'xsmall' | 'small' | 'medium' | 'large'

  /**
   * The stroke of the component. Defaults to black.
   */
  stroke?: string
}

export default function FilterIcon({
  fill = shadeWhite,
  size = 'small',
  stroke = shadeBlack,
  ...other
}: FilterIconProps): React.ReactElement {
  return (
    <FilterBase xmlns="http://www.w3.org/2000/svg" size={size} viewBox="0 0 44 44" {...other}>
      <title>Filter icon</title>
      <desc>Filter icon</desc>
      <path
        d="M0 5a5 5 0 0 1 5-5h34a5 5 0 0 1 5 5v34a5 5 0 0 1-5 5H5a5 5 0 0 1-5-5V5Z"
        fill={fill}
      />
      <path
        stroke={stroke}
        strokeWidth="1.5"
        strokeLinecap="round"
        strokeLinejoin="round"
        d="M13.584 28.38h16.833"
      />
      <circle cx="27.562" cy="28.666" r="1.583" fill={fill} stroke={stroke} strokeWidth="1.5" />
      <path
        stroke={stroke}
        strokeWidth="1.5"
        strokeLinecap="round"
        strokeLinejoin="round"
        d="M13.584 15.047h16.833"
      />
      <path
        d="M27.562 16.917a1.583 1.583 0 1 0 0-3.167 1.583 1.583 0 0 0 0 3.167Z"
        fill={fill}
        stroke={stroke}
        strokeWidth="1.5"
      />
      <path
        stroke={stroke}
        strokeWidth="1.5"
        strokeLinecap="round"
        strokeLinejoin="round"
        d="M30.418 21.714H13.585"
      />
      <path
        d="M16.44 23.584a1.583 1.583 0 1 1 0-3.167 1.583 1.583 0 0 1 0 3.167Z"
        fill={fill}
        stroke={stroke}
        strokeWidth="1.5"
      />
    </FilterBase>
  )
}

export const FilterBase = styled.svg<FilterIconProps>`
  display: flex;
  justify-content: center;
  align-items: center;
  transition: transform 0.4s;
  height: ${(p) => getSvgSize(p.size)};
  width: ${(p) => getSvgSize(p.size)};
`
