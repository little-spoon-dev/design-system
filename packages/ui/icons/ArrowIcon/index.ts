export { default } from '@littlespoon/icons/lib/ArrowIcon'
export * from '@littlespoon/icons/lib/ArrowIcon'
