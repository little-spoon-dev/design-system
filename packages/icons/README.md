# @littlespoon/icons

[![NPM version](https://img.shields.io/npm/v/@littlespoon/icons.svg)](https://www.npmjs.com/package/@littlespoon/icons)

## Install

[Yarn](https://yarnpkg.com/package/@littlespoon/icons):

```sh
yarn add @littlespoon/icons
```

[npm](https://www.npmjs.com/package/@littlespoon/icons):

```sh
npm install @littlespoon/icons --save
```

[UNPKG](https://unpkg.com/browse/@littlespoon/icons/)

## Usage

Import icons:

```tsx
import icons from '@littlespoon/icons'
icons.ArrowIcon
icons.FilterIcon
```

Import arrow icon:

```tsx
import ArrowIcon from '@littlespoon/icons/lib/ArrowIcon'
import FilterIcoon from '@littlespoon/icons/lib/FilterIcon'
```
