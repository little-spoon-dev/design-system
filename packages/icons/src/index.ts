import ArrowIcon from './ArrowIcon'

export { ArrowIcon }

export default {
  ArrowIcon,
}
