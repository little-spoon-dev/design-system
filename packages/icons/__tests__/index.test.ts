import icons, { ArrowIcon as arrowIcon, FilterIcon as filterIcon } from '../src/'
import ArrowIcon from '../src/ArrowIcon'
import FilterIcon from '../src/FilterIcon'

it('matches arrow icon', () => {
  expect(icons.ArrowIcon).toBe(ArrowIcon)
  expect(arrowIcon).toBe(ArrowIcon)
})

it('matches filter icon', () => {
  expect(icons.FilterIcon).toBe(FilterIcon)
  expect(filterIcon).toBe(FilterIcon)
})
