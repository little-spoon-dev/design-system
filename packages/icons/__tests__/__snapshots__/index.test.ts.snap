// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches icons 1`] = `
Object {
  "ArrowIcon": [Function],
}
`;
